README.md

```diff
@@ -0,0 +1,97 @@
+# FP-Signature
+
+Fynd platform's signature logic
+
+
+
+FP-Signature is a versatile npm package that provides signature logic for Fynd platform requests. This package supports both CommonJS and ES modules, and it also comes with a web bundle for direct usage in browsers.
+
+## Installation
+
+You can install FP-Signature via npm:
+
+```bash
+npm install @gofynd/fp-signature
+```
+
+## How to Use
+
+### For CommonJS
+
+```javascript
+const { sign } = require("@gofynd/fp-signature");
+```
+
+### For ES Modules
+
+```javascript
+import { sign } from "@gofynd/fp-signature";
+```
+
+### For Browser
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@gofynd/fp-signature"></script>
+<script>
+    // FP-Signature library will be attached to the global window object
+    FPSignature.sign();
+</script>
+```
+
+## Functionality
+
+### `sign` function
+
+The `sign` function is used to add a signature to a request. It takes two parameters: `request` and `options`.
```

Comment:
Change wordings here it will return generated signature

Reply:
changed line